utils/transaction-parser.ts

```diff
@@ -289,7 +312,9 @@ function parseZetaInstruction(ix: PartiallyDecodedInstruction): Instruction {
       // size: number;
       let liquidateData = decodedIx.data as zetaTypes.liquidate;
       decodedIx.data = {
-        size: utils.convertNativeLotSizeToDecimal(liquidateData.size)
+        size: utils.convertNativeLotSizeToDecimal(
```

Comment:
@xjcaa confirming this is correct?

Reply:
This is the same way that the placeOrder (and V2 and V3) parser is doing it for `size`

Finding it a bit hard to get a liquidation on devnet to verify though :( I used 2 accounts to buy and sell futures respectively, then withdrew as much funds as possible, and tried to liquidate them later but they still have enough margin 

Any way I can quickly verify this? :D

Reply:
You could probably send a dummy liquidation instruction and even if it fails it would show up

Reply:
But yeah think it's fine